tests/integration/src/test/scala/mesosphere/marathon/integration/SharedMemoryIntegrationTest.scala

```diff
@@ -0,0 +1,232 @@
+package mesosphere.marathon
+package integration
+
+import mesosphere.marathon.core.health.{MesosHttpHealthCheck, PortReference}
+import mesosphere.marathon.core.pod.{HostNetwork, MesosContainer, PodDefinition}
+import mesosphere.marathon.integration.facades.AppMockFacade
+import mesosphere.marathon.integration.setup.{BaseMarathon, EmbeddedMarathonTest, MesosConfig}
+import mesosphere.marathon.raml.{Pod, Raml}
+import mesosphere.marathon.state.{HostVolume, VolumeMount}
+import mesosphere.{AkkaIntegrationTest, WhenEnvSet}
+import org.scalatest.time.{Seconds, Span}
+import play.api.libs.json.Json
+
+import scala.collection.immutable.Seq
+
+class SharedMemoryIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonTest {
+
+  val projectDir: String = sys.props.getOrElse("user.dir", ".")
+  override lazy val mesosConfig = MesosConfig(
+    launcher = "linux",
+    containerizers = "docker,mesos",
+    isolation = Some("filesystem/linux,docker/runtime,namespaces/ipc"),
+    imageProviders = Some("docker")
+  )
+
+  override implicit lazy val patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(30, Seconds), interval = Span(1, Seconds))
+
+  "get correct shm size from pod" taggedAs WhenEnvSet(envVarRunMesosTests, default = "true") in {
+    Given("a pod with a single task and a volume")
+    val projectDir = sys.props.getOrElse("user.dir", ".")
```

Comment:
Maybe we should extract this out to a constant so that we don't have to repeat this string?

Reply:
It's actually already extracted, and then repeated in both tests ...